docs/reference/api/docker_remote_api.md

```diff
@@ -124,6 +123,11 @@ This section lists each version from latest to oldest.  Each listing includes a
 * `GET /images/json` now supports filters `since` and `before`.
 * `POST /containers/(id or name)/start` no longer accepts a `HostConfig`.
 * `POST /images/(name)/tag` no longer has a `force` query parameter.
+* `GET /containers/json` has been renamed to `/containers`.
```

Comment:
We probably want to say the the old ones are deprecated and will not be available starting with version X :angel: